CHANGES.md

```diff
@@ -16,6 +16,7 @@ We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (
   the version accordingly. ([#1804](https://github.com/diffplug/spotless/issues/1804)).
 ### Fixed
 * Added support for plugins when using Prettier version `3.0.0` and newer. ([#1802](https://github.com/diffplug/spotless/pull/1802))
+* Added support to run with Java 21 ([#1774](https://github.com/diffplug/spotless/issue/1774)).
```

Comment:
https://github.com/diffplug/spotless/issues/1774

issue -> issues 😄

Reply:
still is `issue` - should be `issues` in link 😄

Reply:
I should learn more about merges ;-) thx Slawek